digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/pointasset/PavedRoadService.scala

```diff
@@ -124,7 +124,7 @@ class PavedRoadService(roadLinkServiceImpl: RoadLinkService, eventBusImpl: Digir
     (expiredAssetsIds, newAndUpdatedAssets)
   }
 
-  override protected def updateWithoutTransaction(ids: Seq[Long], value: Value, username: String, vvhTimeStamp: Option[Long] = None, sideCode: Option[Int] = None, measures: Option[Measures] = None, informationSource: Option[Int] = None): Seq[Long] = {
+  override def updateWithoutTransaction(ids: Seq[Long], value: Value, username: String, vvhTimeStamp: Option[Long] = None, sideCode: Option[Int] = None, measures: Option[Measures] = None, informationSource: Option[Int] = None): Seq[Long] = {
```

Comment:
Why is this no longer a protected method?

Reply:
Because we need to use updateWithoutTransaction method in other places.